hawkular-wildfly-agent-feature-pack/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml

```diff
@@ -609,7 +609,10 @@
         <operation-dmr name="JDR"      operation-name="generate-jdr-report" path="/subsystem=jdr" />
         <operation-dmr name="Reload"   operation-name="reload" />
         <operation-dmr name="Resume"   operation-name="resume" />
-        <operation-dmr name="Shutdown" operation-name="shutdown" /> <!-- shutdown and restart -->
+        <operation-dmr name="Shutdown" operation-name="shutdown" > <!-- shutdown and restart -->
```

Comment:
On 12 Jul 2016, at 13:45, John Mazzitelli wrote:

>> @@ -609,7 +609,10 @@
>>          <operation-dmr name="JDR"      
>> operation-name="generate-jdr-report" path="/subsystem=jdr" />
>>          <operation-dmr name="Reload"   operation-name="reload" />
>>          <operation-dmr name="Resume"   operation-name="resume" />
>> -        <operation-dmr name="Shutdown" operation-name="shutdown" /> 
>> <!-- shutdown and restart -->
>> +        <operation-dmr name="Shutdown" operation-name="shutdown" > 
>> <!-- shutdown and restart -->
>
> @pilhuhn should we take the opportunity now and rename 
> "operation-name" to something less confusing like "internal-name" ?

I would appreciate this. Bonus: it is transparent for users/clients.